eng/tox/create_package_and_install.py

```diff
@@ -165,21 +173,39 @@ def build_and_discover_package(setuppy_path, dist_dir, target_setup, package_typ
         default="wheel",
     )
 
+    parser.add_argument(
+        "--pre-download-disable",
+        dest="pre_download",
+        help="During a dev build, we will restore package dependencies from a dev feed before installing them. The presence of this flag disables that behavior.",
+        action="store_true",
+    )
 
     args = parser.parse_args()
 
+    commands_options = []
+    built_pkg_path = ""
     setup_py_path = os.path.join(args.target_setup, "setup.py")
+    additional_downloaded_reqs = []
+    tmp_dl_folder = os.path.join(args.distribution_directory, "dl")
+    os.mkdir(tmp_dl_folder)
+
+    # If extra index URL is passed then set it as argument to pip command
+    if args.extra_index_url:
```

Comment:
Perhaps I'm missing something but shouldn't we no longer need to pass the extra-index-url after this update?